src/components/Button/Button.tsx

```diff
@@ -2,13 +2,26 @@ import React from "react";
 import "./style.scss";
 
 interface ButtonProps extends React.PropsWithChildren {
-
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  disabled?: boolean;
+  className?: string;
 }
 
-const Button: React.FC<ButtonProps> = ({ children }) => {
+const Button: React.FC<ButtonProps> = ({
+  children,
+  onClick,
```

Comment:
Instead of destructuring props you can use ...props here for these props and use {...props} in button element

Reply:
I mean disabled and onClick props

Reply:
Done.